### DISCUSSION TITLE
AdonisJS V5: JSDOM HTML question about testing

### DISCUSSION THREAD

**Discussion by menzhessarov** [adonisjs/core#1339] [Help]
Guys, how do I go about testing the basic login html page showcased in V5 docs? My initial idea is to insert username and password to input tags and then after redirect check cookies. But Idk what will happen to jsdom when I submit the form, will it redirect just like in the browser?

**Comment by McSneaky** [comment]
If I recall it correctly JSDOM follows redirects by default
